packages/client/src/components/atoms/AddToCalendar/AddToCalendar.tsx

```diff
@@ -0,0 +1,106 @@
+import React from "react";
+import { useTranslation, ActionButton } from "@eisbuk/translations";
+import { useSelector } from "react-redux";
+import { DateTime } from "luxon";
+import { ICalendar } from "datebook";
+
+import makeStyles from "@mui/styles/makeStyles";
+
+import Button from "@mui/material/Button";
+
+import { LocalStore } from "@/types/store";
+import { __addToCalendarButtonId__ } from "@/__testData__/testIds";
+import { getAboutOrganization } from "@/store/selectors/app";
+
+interface Props {
+  /**
+   * Record of subscribed slots with subscribed slotIds as keys and subscribed duration as value.
+   * Doesn't need to be organized as we're checking for each value by key (no need for ordering and grouping).
+   */
+  bookedSlots?: LocalStore["firestore"]["data"]["bookedSlots"];
+}
+const AddToCalendar: React.FC<Props> = ({ bookedSlots = {} }) => {
+  const classes = useStyles();
+
+  const { displayName = "", location = "" } =
+    useSelector(getAboutOrganization) || {};
+  const { t } = useTranslation();
+
+  // is displayed when bookings are finalized aka deadline is null
```

Comment:
I don't understand this...I understand it might be an explanation of the component, but then it should be in component JSDoc..

Reply:
Or just removed to avoid confusion if it was left here by accident